trace_context/README.md

```diff
@@ -0,0 +1,28 @@
+# Trace Context
+
+The trace context positions an incoming request in potentially multiple trace
+graphs. For example, an incoming request might have information about its
+gateway in one service, and information from its direct upstream in another.
+
+Two propagation fields carry the common and vendor-specific properties that
+make up the trace context.
+
+* `Trace-Parent` describes the position of the incoming request in its trace graph in a portable, fixed-length format. Its design focuses on fast parsing.
+
+* `Trace-State` maps all graphs the incoming parent is a part of in potentially vendor-specific formats. For example, if an ancestor of this request was in a different trace, there will be a separate entry for the last position it was in that trace graph.
+
+Notably, the `Trace-State` field is unreliant on data in the `Trace-Parent`,
+except if the direct upstream is fully described by `Trace-Parent`.
```

Comment:
Maybe I'm missing something here, but how could a user tell whether the "direct upstream is fully described by `Trace-Parent`"? Specially in the case of having more than one trace graph involved. 

Also, in the case of multiple trace graphs is there any rule regarding which of the identifiers should stay in the `traceparent` header? Are implementors allowed to "move" a different set of identifiers between the parent and state headers? e.g. `traceparent` has yelp-generated identifiers but as soon as it crosses to another vendor, they might want to propagate yelp's identifiers in `tracestate` and have their own in `traceparent`.

Reply:
I've removed the ability to split across headers (as this was copy/paste from correlation context) and opened an issue in case folks want to revisit later  https://github.com/w3c/distributed-tracing/issues/80